Tests/MapboxNavigationTests/Support/AccessToken.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import MapboxDirections
+
+extension Directions {
+    static var mocked: Directions {
+        return .init(credentials: .mocked)
+    }
+}
+
+extension DirectionsCredentials {
+    static var mocked: DirectionsCredentials {
+        return .init(accessToken: .mockedAccessToken, host: nil)
+    }
+}
+
+extension String {
+    static var mockedAccessToken: String { "any" }
```

Comment:
This works, though it’s possible that lower-level libraries may introduce more rigorous client-side validation of access tokens in the future. MapboxDirectionsTests uses [a cheeky placeholder access token](https://github.com/mapbox/mapbox-directions-swift/blob/60b4a028119cef727648d981d2daccea0641138c/Tests/MapboxDirectionsTests/DirectionsTests.swift#L14-L15) that might be easier to spot while debugging.

Reply:
Thank you for your suggestion, I will change from "any" to "pk.feedCafeDadeDeadBeef-BadeBede.FadeCafeDadeDeed-BadeBede" and reuse "BogusToken" name convention.

Reply:
Fixed in https://github.com/mapbox/mapbox-navigation-ios/pull/3029/commits/473c67c8dcd37037e0b72ff3311ddcc375a8007e